File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/GetChannels.java
package me.grishka.houseclub.api.methods;

import java.util.List;

import me.grishka.houseclub.api.ClubhouseAPIRequest;
import me.grishka.houseclub.api.model.Channel;

public class GetChannels extends ClubhouseAPIRequest<GetChannels.Response> {
    public GetChannels() {
        super("GET", "get_channels", Response.class);
    }

    public static class Response {
        public List<Channel> channels;
    }
}

File: Houseclub/src/main/java/me/grishka/houseclub/api/methods/GetChannels.kt
package me.grishka.houseclub.api.methods

import me.grishka.houseclub.api.ClubhouseAPIRequest
import me.grishka.houseclub.api.model.Channel

class GetChannels : ClubhouseAPIRequest<GetChannels.Response?>("GET", "get_channels", Response::class.java) {
    class Response {
        var channels: List<Channel>? = null
    }
}